vdr/didjwk/utils.go

```diff
@@ -0,0 +1,36 @@
+package didjwk
+
+import (
+	"fmt"
+	"reflect"
+
+        "github.com/lestrrat-go/jwx/jwk"
+)
+
+// rawPrivateKeyOf returns the private key component of a jwk.Key, or nil if one is not available (e.g. public key only JWK's). An error is returned if a public key is not contained in the JWK.
+func rawPrivateKeyOf(key jwk.Key) (any, error) {
```

Comment:
test coverage is low because there's no associated `_test.go` for this file. I'd say just move it to `resolver.go`